organization/external-communications.md

```diff
@@ -0,0 +1,127 @@
+1. Fork the pyOpenSci website repo: [https://github.com/pyOpenSci/pyopensci.github.io](https://github.com/pyOpenSci/pyopensci.github.io).
+2. Work on your blog post locally. Be sure to include alt text as needed, and create an author page if you do not already have one.
+3. Submit a pull request from your fork of the website repo to [https://github.com/pyOpenSci/pyopensci.github.io](https://github.com/pyOpenSci/pyopensci.github.io).
+4. Make sure that the blog post passes CI checks that look for spelling errors, broken links and more. Please note that occasionally the `htmlproofer` check will fail because the URL does not exist yet. We have implemented a fix for this, but it occasionally does not work.
+5. Request review of the blog post. A member of the pyOpenSci staff should always review any new content being added to the website.
+6. Once you have an approving review, the pull request can be merged. In the case of guest blog posts, a member of the pyOpenSci team will merge your post.
+
+pyOpenSci has GitHub organization-wide policies in place that ensure all pull requests are generally reviewed prior to being merged. In some instances we may bypass those if the PR is a small fix or critical update. In most cases a review is preferred. Please contact the Executive Director and Founder, Leah Wasser, with any questions or needs for clarification.
+
+**A note on guest blog posts**
+
+Anyone who has had their package accepted into the pyOpenSci ecosystem is encouraged to submit a blog post (or series of blog posts) about their review experience and/or their package. When submitting your blog post, please include the following YAML elements:
+
+```
+layout: single
+title: "The title of your blog post"
+excerpt: "One to five sentences summarizing your post. This text will appear on our [blog landing page](https://pyopensci.org/blog)"
+author: "Your name"
+permalink: /blog/url-for-your-post.html
+header:
+  overlay_image: images/headers/your-blog-post-header-image.png
+  overlay_filter: rgba(20, 13, 36, 0.8)
+categories:
+  - blog-post
+  - community
+classes: wide
+toc: true
+comments: true
+---
+```
+
+For more information on how pyOpenSci uses GitHub, please refer to the [GitHub section of our handbook](https://www.pyopensci.org/handbook/community/github/intro.html).
+
+### Promoting blog posts
+
+All blog posts should be promoted, regardless of whether or not they were written by a guest or a pyOpenSci employee! For promotion, coordinate with the Community Manager to ensure that there’s a tailored message for each of the following platforms:
+* Slack
+* Discourse
+* BlueSky
+* LinkedIn
+* Fosstodon
+
+In addition to pyOpenSci promotions, we monitor social media for any personal posts authors have created to promote their post, boost accordingly.
+
+## Newsletters
+
+We publish two newsletters:
+* A weekly newsletter on LinkedIn, published on Thursdays
+* A monthly newsletter published on LinkedIn and sent to all Mailchimp subscribers
```

Comment:
Flaggin mailchimp. If we made a decision after august 14 that i've forgotten you can remind me!